StudioCore/MapStudioNew.cs

```diff
@@ -715,6 +689,76 @@ private void Update(float deltaseconds)
+                if (ImGui.BeginMenu("Settings"))
+                {
+                    if (_projectSettings == null || _projectSettings.ProjectName == null)
+                    {
+                        ImGui.MenuItem("Project Settings: (No project)", false);
+                    }
+                    else
+                    {
+                        if (ImGui.BeginMenu($@"Project Settings: {_projectSettings.ProjectName}", Editor.TaskManager.GetLiveThreads().Count == 0))
+                        {
+                            bool useLoose = _projectSettings.UseLooseParams;
+                            if ((_projectSettings.GameType == GameType.DarkSoulsIISOTFS || _projectSettings.GameType == GameType.DarkSoulsIII) && ImGui.Checkbox("Use Loose Params", ref useLoose))
+                            {
+                                _projectSettings.UseLooseParams = useLoose;
+                            }
+                            bool usepartial = _projectSettings.PartialParams;
+                            if (_projectSettings.GameType == GameType.EldenRing && ImGui.Checkbox("Partial Params", ref usepartial))
+                            {
+                                _projectSettings.PartialParams = usepartial;
+                            }
+                            ImGui.EndMenu();
+                        }
+                    }
+                    if (ImGui.MenuItem("Enable Texturing (alpha)", "", CFG.Current.EnableTexturing))
+                    {
+                        CFG.Current.EnableTexturing = !CFG.Current.EnableTexturing;
+                    }
+                    if (ImGui.BeginMenu("Viewport Settings"))
+                    {
+                        if (ImGui.Button("Reset"))
+                        {
+                            CFG.Current.GFX_Camera_FOV = CFG.Default.GFX_Camera_FOV;
+
+                            _msbEditor.Viewport.FarClip = CFG.Default.GFX_RenderDistance_Max;
+                            CFG.Current.GFX_RenderDistance_Max = _msbEditor.Viewport.FarClip;
+
+                            _msbEditor.Viewport._worldView.CameraMoveSpeed_Slow = CFG.Default.GFX_Camera_MoveSpeed_Slow;
+                            CFG.Current.GFX_Camera_MoveSpeed_Slow = _msbEditor.Viewport._worldView.CameraMoveSpeed_Slow;
+
+                            _msbEditor.Viewport._worldView.CameraMoveSpeed_Normal = CFG.Default.GFX_Camera_MoveSpeed_Normal;
+                            CFG.Current.GFX_Camera_MoveSpeed_Normal = _msbEditor.Viewport._worldView.CameraMoveSpeed_Normal;
+
+                            _msbEditor.Viewport._worldView.CameraMoveSpeed_Fast = CFG.Default.GFX_Camera_MoveSpeed_Fast;
+                            CFG.Current.GFX_Camera_MoveSpeed_Fast = _msbEditor.Viewport._worldView.CameraMoveSpeed_Fast;
+                        }
+                        float cam_fov = CFG.Current.GFX_Camera_FOV;
+                        if (ImGui.SliderFloat("Camera FOV", ref cam_fov, 40.0f, 140.0f))
```

Comment:
I will probably move these eventually to per viewport settings but should be fine for now.